demo/date-picker-localization-demos.html

```diff
@@ -50,6 +48,43 @@ <h3>Localizing</h3>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Custom key input</h3>
```

Comment:
I wonder if this demo should be under "Localization". 🤔

Reply:
Not done because it was decided by Jouni.